Classes/UIScrollView+InfiniteScroll.m

```diff
@@ -34,7 +34,7 @@ static void PBSwizzleMethod(Class c, SEL original, SEL alternate) {
  *
  *  See https://github.com/pronebird/UIScrollView-InfiniteScroll/issues/31
  *
- *  @param tableView
+ *  @param tableView A helper function to force table view to update its content size.
```

Comment:
Hi @musbaalbaki,

Thanks for your PR. But the parameter description does not reflect the parameter itself. In that case it's tableView which is not a function.